### ISSUE TITLE
AnimeFenix | www.animefenix.tv

### ISSUE THREAD

**Issue by PreMiD-Automations** [PreMiD/Activities#5427]
### Discussed in https://github.com/PreMiD/Presences/discussions/4505

<div type='discussions-op-text'>

<sup>Originally posted by **FALISO** October 18, 2020</sup>
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
PAN#7077
**What's the name of the service?** 
https://www.animefenix.com
**What should the Presence display?** 
basically almost the same as animeflv
**If possible, please provide a logo for the service (512x512 minimum)** 
![AveFenix](https://user-images.githubusercontent.com/73078072/96377710-c1717100-115d-11eb-86be-90123d598942.png)

</div>

**Comment by DarkVillager** [comment]
Domain changed => [https://animefenix.tv/](https://animefenix.tv/)
